GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/GVFSUpgradeReminderTests.cs

```diff
@@ -136,8 +137,7 @@ private bool ServiceLogContainsUpgradeMessaging()
             string serviceLogFolder = Path.Combine(
                 Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData),
                 "GVFS",
-                GVFSServiceProcess.TestServiceName,
-                "Logs");
+                GVFSServiceProcess.TestServiceName);

```

Comment:
Why was "Logs" removed from the path?  If we keep a subfolder for log files it might be easier to adjust the ACLs for log files (assuming that's something that is still required).

Reply:
>>Why was "Logs" removed from the path?

After moving the secure files (`gvfs.config` and Upgrader files) only the log files remained in the old location (`C:\ProgramData\GVFS`).

But your thought about ACLs make sense. Also in future we can use this directory to keep non-log files as well.

Adding "Logs" subdirectory - [commit](https://github.com/microsoft/VFSForGit/pull/1597/commits/e052d104edc4a4fcca6bd822f2573d37049b5eb2)